ui/src/app/tasks-jobs/executions/executions.component.html

```diff
@@ -88,6 +88,13 @@ <h1>{{ 'executions.main.title' | translate }}</h1>
     <ng-container>{{ 'executions.main.exitCode' | translate }}</ng-container>
     <clr-dg-filter style="display: none"></clr-dg-filter>
   </clr-dg-column>
+  <clr-dg-column
+    (clrDgColumnResize)="updateContext('sizeStatus', $event)"
+    [style.width.px]="context.sizeStatus | datagridcolumn: datagrid:contextName:15 | async"
+  >
+    <ng-container>{{ 'commons.status' | translate }}</ng-container>
```

Comment:
Does the ` | translate` handle i18n? (eg. `successo` when in Italian)

Reply:
@onobc  The `| translate` handles the i18n, correct. 
On this specific line ( 95 ) commons.status stands for the string equivalent to `Status` in english.
If you take a look on the data table this text should be displayed within the table header as a column indicator ( header ) for the below executions statuses ( cells ).
Let me know if this clarifies better the context, thanks !

Reply:
Sounds good @claudiahub , thanks for the clarification.